### PR TITLE
added debugging subcomponents

### PR DESCRIPTION
**Pull request by chirp1**

### CODE REVIEW

**Comment by chirp1** [comment]
Hi Tim, please review my updates for issue #270. Thanks!

**Comment by chirp1** [comment]
Hi Tim, would you review my updates for issue #270 and give me your feedback? Thanks!

File: ref/general/reference/troubleshooting.adoc [hunk 1]
```diff
@@ -147,3 +147,231 @@ If the data does not show a `Ready` state and the problem is persistent, there m
...
+
+To obtain the logs, enter the following command:
+
+----
+oc -n kabanero logs $(oc -n kabanero get pods -l name=kabanero-operator-admission-webhook -o=jsonpath={.items[0].metadata.name}
+----
+
+=== appsody
+
+The appsocy component manages the deployment of the application container.
```
Comment (kaczyns) [inline ref/general/reference/troubleshooting.adoc]: appsocy -> appsody

File: ref/general/reference/troubleshooting.adoc [hunk 2]
```diff
@@ -147,3 +147,231 @@ If the data does not show a `Ready` state and the problem is persistent, there m
...
+oc -n openshift-operators describe pods -l name=appsody-operator
+----
+
+To obtain the logs, enter the following command:
+
+----
+oc -n openshift-operators logs $(oc -n openshift-operators get pods -l name=appsody-operator -o=jsonpath={.items[0].metadata.name})
+----
+
+=== che
```
Comment (kaczyns) [inline ref/general/reference/troubleshooting.adoc]: This one is fine today, but it's changing to CodeReady Workspaces soon.  kabanero-io/roadmap#127

File: ref/general/reference/troubleshooting.adoc [hunk 3]
```diff
@@ -147,3 +147,231 @@ If the data does not show a `Ready` state and the problem is persistent, there m
 . Ensure that `webhooks-extension-sink` KService is in the `Ready` state.
    - Use the `oc get kservice -n tekton-pipelines` command.
    - If the `webhooks-extension-sink` KService is not in the `Ready` state, delete a revision object that is associated with the `webhooks-extension-sink`. For example, use the `oc delete rev -l serving.knative.dev/configuration=webhooks-extension-sink` command.
+
```
Comment (kaczyns) [inline ref/general/reference/troubleshooting.adoc]: This is not related to this PR at all, but the whole section starting at line 85 to this point is no longer valid.  The tekton dashboard webhook extension does not use Knative anymore.  We should probably ask Adam Roberts for an updated debugging section.

File: ref/general/reference/troubleshooting.adoc (hunk 1 above)
Comment (chirp1) [reply ref/general/reference/troubleshooting.adoc]: OK. made change

File: ref/general/reference/troubleshooting.adoc (hunk 3 above)
Comment (chirp1) [reply ref/general/reference/troubleshooting.adoc]: Thanks for the info. I slacked with Adam to find out what to do. He indicated that the troubleshooting still applies to customers on releases before 0.3.0., and so we should leave this info in. As far as 0.3.0 and later, he indicated that Webhook creation is now really reliable. So, at this point he doesn't have a troubleshooting tip to add.

File: ref/general/reference/troubleshooting.adoc (hunk 2 above)
Comment (chirp1) [reply ref/general/reference/troubleshooting.adoc]: OK do you have the blurb for the CodeReady Workspaces that is replacing the che component? If you do, I could update the troubleshooting

Comment (kaczyns) [reply ref/general/reference/troubleshooting.adoc]: On line 247 can change `che` to `CodeReady Workspaces`.
on line 252 change `app=che-operator` to `app=codeready-operator`
on line 258 change `app=che-operator` to `app=codeready-operator`

I would have given you this info in my comment earlier but the code was not merged yet.  It's merged now.

Comment (chirp1) [reply ref/general/reference/troubleshooting.adoc]: Thanks for the info. I've made those changes. 

- I believe I would also change line 245 from `che` to `CodeReady Workspaces` to change the component name. Is that right? 

- Also, would I change some of the 170-179 lines as they mention the che component? If so, what should I change them to?

Comment (kaczyns) [reply ref/general/reference/troubleshooting.adoc]: Yes 245 would change also.  Right 170-180 would change too, it should look like this:
```yaml
codereadyWorkspaces:
  ready: True
  message: Error message here
  operator:
    version: 2.0.0
    instance:
      devfileRegistryImage: repository:tag
      cheWorkspaceClusterRole: role-name
      openShiftOAuth: false
      selfSignedCert: false
      tlsSupport: false
```

Comment (chirp1) [reply ref/general/reference/troubleshooting.adoc]: Hi Tim, Thanks for the info. I made the updates. Would you take a look and let me know whether the updates look OK?

Comment (kaczyns) [reply ref/general/reference/troubleshooting.adoc]: Hi Karen, the updates look fine to me.  Thanks!

**Pull request merged**